test/e2e/testdata/apisix-gw-config-default.yaml

```diff
@@ -59,6 +59,7 @@ apisix:
 
   allow_admin:                  # http://nginx.org/en/docs/http/ngx_http_access_module.html#allow
     - 127.0.0.0/24              # If we don't set any IP list, then any IP access is allowed by default.
+    - 0.0.0.0/0
```

Comment:
Should remove this.  Had add this in `test/e2e/testdata/apisix-gw-config.yaml` already.

Reply:
OK.

Reply:
Fixed.